backend/experiment/actions/consent.py

```diff
@@ -20,8 +21,9 @@ class Consent(BaseAction):  # pylint: disable=too-few-public-methods
                 contentiones, vix ex maiorum denique! Lorem ipsum dolor sit \
                 amet, nec te atqui scribentur. Diam molestie posidonium te sit, \
                 ea sea expetenda suscipiantur contentiones."
-    
-    def __init__(self, text=default_text, title='Informed consent', confirm='I agree', deny='Stop'):
+
+    def __init__(self, text=default_text, title='Informed consent', confirm='I agree', deny='Stop', style=FrontendStyle()):
+        super(Consent, self).__init__(style)
```

Comment:
You can just write `super().__init__(style)` here.

Reply:
```suggestion
        super().__init__(style)
```